aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -316,6 +318,13 @@ struct Vec256 {
     }
     return ret;
   }
+  Vec256<T> copysign(const Vec256<T> &sign) const {
+    Vec256<T> ret;
+    for (int i = 0; i < size(); i++) {
```

Comment:
type `i` like `size()`

Reply:
I've already changed it. `size()` is `int`.